IPython/core/ultratb.py

```diff
@@ -315,16 +315,6 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
-if py3compat.PY3:
-    fixed_getargvalues = inspect.getargvalues
-else:
-    # Fixes for https://github.com/ipython/ipython/issues/8293
-    #       and https://github.com/ipython/ipython/issues/8205.
-    # The relevant bug is caused by failure to correctly handle anonymous tuple
-    # unpacking, which only exists in Python 2.
-    fixed_getargvalues = with_patch_inspect(inspect.getargvalues)
```

Comment:
OK, then can you add a note to its docstring saying that it's deprecated since IPython 6.0?